### ISSUE TITLE
When "text changed" checkbox is ticked off a blinking frame still shows on page

### ISSUE THREAD

**Issue by sturlath** [kdzwinel/DOMListenerExtension#8]
I have ticked off the yellow  "Text changed"  checkbox to stop having our "Session expires in xx:xx min"  showing in the logs. And it does. But there is still a blinking frame on the page Im looking at. 
It would be great to let the box change.

One other idea would be to be able to stop the logging/blinking on just that element but still get all the others.

**Comment by kdzwinel** [comment]
Hey! Thanks for input. So you are saying that nodes that do not match current filter should not be highlighted? What do you mean by "It would be great to let the box change." ?

**Comment by sturlath** [comment]
Yes I would think that if you un-tick for example "Text Changed" I should not see a blinking (highlighted)  frame on the page where text is changing. Maby you should have a "un-monitored changes number/list" that does not show up in the log or on page.

But even better would be for me to be able to select the elements I don´t want to monitor. I would like to have "Text Changed" in the logg and have it blinking when text is changed on the page. But not for things like the "clock label" that blinks and logs every second.

What I meant by the "It would... change" comment I can´t remember :-) Ill let you know if it pops up again.
